audit-opentelemetry/chart/templates/tests/test-opentelemetry-config.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.testFramework.enabled -}}
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-test
+  namespace: {{ .Release.Namespace }}
+  labels:
+    type: integration-test
+    {{- include "plugin.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded"
+data:
+  run.sh: |-
+
+    #!/usr/bin/env bats
+
+    load "/usr/lib/bats/bats-detik/utils"
+    load "/usr/lib/bats/bats-detik/detik"
+    
+    DETIK_CLIENT_NAME="kubectl"
+    
+    @test "Verify successful deployment and running status of the {{ .Release.Name }}-operator pod" {
+        verify "there is 1 deployment named '{{ .Release.Name }}-operator'"
+        try "at most 4 times every 5s to get pods named '{{ .Release.Name }}-operator-[a-z0-9]\+-' and verify that '.status.phase' is 'running'" 
+    }
+
+    @test "Verify successful deployment of the {{ .Release.Name }}-operator service" {
+        verify "there is 1 service named '^{{ .Release.Name }}-operator\$'"
+    }
+
+    {{- if .Values.openTelemetry.logsCollector.enabled }}
+    @test "Verify successful deployment and running status of the audit-logs-collector" {
+        try "at most 5 times every 10s to get pods named 'audit-logs-collector' and verify that '.status.phase' is 'running'"
+    }
+    {{- end -}}
+
+    {{- if .Values.openTelemetry.metricsCollector.enabled }}
+    @test "Verify successful deployment and running status of the metrics-collector" {
+        try "at most 5 times every 10s to get pods named 'metrics-collector' and verify that '.status.phase' is 'running'"
```

Comment:
See other comment in metrics-collector.yaml: Rename this to audit-logs-metrics or something unique